### ISSUE TITLE
Enhance map design

### ISSUE THREAD

**Issue by martinheppner** [bahnzumberg/zuugle-suchseite#318]
- [x] Move map up
- [x] Move number of results below map
- [x] Fix black button (text and behaviour)

![Image](https://github.com/bahnzumberg/zuugle-suchseite/assets/105971399/97efca2a-a79e-437c-a4ec-eb91f6b383e2)

**Comment by Falsal** [comment]
Kachelansicht instead of Remove it
send the svg to Martin

**Comment by martinheppner** [comment]
Translation files updated into dev branch.

New Icon created in \src\icons\svg\ called list.svg, which should be used, when "kartenansicht_entfernen" is shown.

**Comment by Falsal** [comment]
@martinheppner :  2 points regarding translations for map buttons.

1) Please do not use spaces or special characters in translation key names: 
I have received these 2 keys : "start / main pages  and "main only", Tried different ways, but I am unable to use them at the moment. please use the variation that i entered in the Google sheet to export new updated files

2) Some names are too long , please make them shorter (suggestion: remove the word 'GO to' ) English example instead of "Go to map" lets say "View Map" and "View List" and so on.



Some background : 
https://poeditor.com/blog/translation-keys/ 
Excerpt:
" Avoid spaces and special characters
Use alphanumeric characters and underscores to ensure compatibility and ease of use across different systems or platforms. For example, use home_page_title instead of home page title."

**Comment by martinheppner** [comment]
I do not know what you are refering to. I just did the translations in existing lines. I did not change the key.

![image](https://github.com/bahnzumberg/zuugle-suchseite/assets/105971399/0f90d6ce-6feb-4ca8-9d53-dc231b69ac81)

**Comment by Falsal** [comment]
@martinheppner :
I changed the keys (values under column "space name") myself, I need now a new set of translation files please.

fyi : these are part of combined key values when we reference them in the code so now we have for example t("main_only.kartenansicht_entfernen") the part "main_only" comes from the column "space name"

**Comment by martinheppner** [comment]
You created the key values. I did not change them. Period. I received a new translation, so I will deploy a new set of translations in a few minutes to dev branch.

**Comment by martinheppner** [comment]
New translations are uploaded

**Comment by Falsal** [comment]
@martinheppner : 
Let us meet earlier before our weekly meeting to fix the translation tables.

Just in case you have time to fix it, here is an explanation as clear as I can make it, 
if not clear I will explain in person this evening.

1) referring to the column names of the translation table on the sheet, (yellow background)
2) we should use column name  **"space name"** as the **source** for our **space names inside the JSON files**
3) it seems you have used the column : "Pagename" instead
4) that is the reason we got now different space names inside the latest JSON files .
5) the translation on the page will not work before we fix this.
6) I will do a manual fix on the translation files until we I get a new improved set of files this evening (or whenever you can provide them)

**Comment by martinheppner** [comment]
Faris, I can not be more clear than this: I did not change anything but fill in translations in existing lines in the German, English, French, Italian and Slovenian column.

I undid your renaming if the page column, because this killed the Python Script, used to export into JSON files.

If there is a problem now, I did not contribute anything to it and I would not know why there should be a problem because of the unchanged JSON structure.
